app/src/main/java/nerd/tuxmobil/fahrplan/congress/MainActivity.java

```diff
@@ -375,7 +375,7 @@ public void openLectureDetail(Lecture lecture, int mDay) {
         MyApp.LogDebug(LOG_TAG, "openLectureDetail sidePane=" + sidePane);
         if (sidePane != null) {
             FragmentManager fm = getSupportFragmentManager();
-            sidePane.setVisibility(View.VISIBLE);
+            fm. popBackStack(FragmentTags.DETAIL, FragmentManager.POP_BACK_STACK_INCLUSIVE);
```

Comment:
Can you please remove the whitespace here? Please amend the change and force push the branch.